.github/workflows/benchmark-build.yaml

```diff
@@ -0,0 +1,459 @@
+          labels: ${{ steps.cdn-broker.outputs.labels }}
+
+      - name: Build and push cdn-marshal docker
+        uses: docker/build-push-action@v6
+        with:
+          context: ./
+          file: ./docker/cdn-marshal.Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.cdn-marshal.outputs.tags }}
+          labels: ${{ steps.cdn-marshal.outputs.labels }}
+
+      - name: Build and push cdn-whitelist docker
+        uses: docker/build-push-action@v6
+        with:
+          context: ./
+          file: ./docker/cdn-whitelist.Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.cdn-whitelist.outputs.tags }}
+          labels: ${{ steps.cdn-whitelist.outputs.labels }}
+
+      - name: Build and push state-relay-server docker
+        uses: docker/build-push-action@v6
+        with:
+          context: ./
+          file: ./docker/state-relay-server.Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.state-relay-server.outputs.tags }}
+          labels: ${{ steps.state-relay-server.outputs.labels }}
+
+      - name: Build and push prover-service docker
+        uses: docker/build-push-action@v6
+        with:
+          context: ./
+          file: ./docker/prover-service.Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.prover-service.outputs.tags }}
+          labels: ${{ steps.prover-service.outputs.labels }}
+
+      - name: Build and push orchestrator docker
+        uses: docker/build-push-action@v6
+        with:
+          context: ./
+          file: ./docker/orchestrator.Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.orchestrator.outputs.tags }}
+          labels: ${{ steps.orchestrator.outputs.labels }}
+
+      - name: Build and push commitment-task docker
+        uses: docker/build-push-action@v6
+        with:
+          context: ./
+          file: ./docker/commitment-task.Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.commitment-task.outputs.tags }}
+          labels: ${{ steps.commitment-task.outputs.labels }}
+
+      - name: Build and push submit-transactions docker
+        uses: docker/build-push-action@v6
+        with:
+          context: ./
+          file: ./docker/submit-transactions.Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.submit-transactions.outputs.tags }}
+          labels: ${{ steps.submit-transactions.outputs.labels }}
```

Comment:
Looks like here I need to update the bin `target/$TARGETARCH/release/submit-transactions` by running `cargo build --release --features benchmarking` as it will be used in `./docker/submit-transactions.Dockerfile`. Is there a place to upload the bin or it will be generated when doing the real run?